docs/testplan/transceiver/system_test_plan.md

```diff
@@ -0,0 +1,263 @@
+| cmis_bootup_low_power_test_supported | boolean | False | ✗ | platform | Whether to test that CMIS transceivers boot up in low power mode when xcvrd is disabled during startup |
+| tx_disable_test_supported | boolean | False | ✗ | transceivers | Whether transceiver supports Tx disable testing and DataPath state verification |
+| optics_si_settings | dict | {} | ✗ | transceivers | Dictionary containing optics SI settings with nested structure for parameters like OutputAmplitudeTargetRx, OutputEqPreCursorTargetRx, OutputEqPostCursorTargetRx, etc. Each parameter contains per-lane values (e.g., OutputAmplitudeTargetRx1-8). Test runs if dictionary is non-empty. |
+| media_si_settings | dict | {} | ✗ | platform_hwsku_overrides | Dictionary containing media SI settings following media_settings.json structure for comparison with APPL_DB values. Test runs if dictionary is non-empty. |
+| frequency_values | list | [] | ✗ | transceivers | List of frequency values for C-CMIS transceivers. First value is the default frequency, followed by test frequencies (min/max supported). Test runs if list is non-empty. |
+| tx_power_values | list | [] | ✗ | transceivers | List of tx power values in dBm for C-CMIS transceivers. First value is the default tx power, followed by test power levels (min/max supported). Test runs if list is non-empty. |
+| expected_application_code | integer | - | ✗ | platform_hwsku_overrides | Expected application code value for the specific transceiver type, platform, and hwsku combination. When defined, the test will verify that the actual application code read from the transceiver matches this expected value. |
+
+For information about attribute override hierarchy and precedence, please refer to the [Transceiver Onboarding Test Plan](../transceiver_onboarding_test_plan.md#test-cases) documentation.
+
+## CLI Commands Reference
+
+For detailed CLI commands used in the test cases below, please refer to the [CLI Commands section](../transceiver_onboarding_test_plan.md#cli-commands) in the transceiver onboarding test plan. This section provides comprehensive examples of all relevant commands
+
+## Test Cases
+
+**Test Execution Prerequisites:**
+
+The following tests from the [Transceiver Onboarding Test Plan](../transceiver_onboarding_test_plan.md#test-cases) will be run prior to executing the system tests:
+
+- Transceiver presence check
+- Ensure active firmware is gold firmware (for non-DAC CMIS transceivers)
+- Link up verification
+- LLDP verification (if enabled)
+
+**Assumptions for the Below Tests:**
+
+- All the below tests will be executed for all the transceivers connected to the DUT (the port list is derived from the `port_attributes_dict`) unless specified otherwise.
+
+## Test Execution Flow
+
+### Recommended Test Order
+
+The following execution order is recommended to minimize system disruption and ensure reliable test results:
```

Comment:
Aren't the tests run serially on the dut? And shouldn't they be independent of each other? I am confused about why we need to specify the desired order of test execution.